### ISSUE TITLE
organize holacratically

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#404]
http://www.holacracy.org/

h/t @JanelleOrsi at #384.

**Comment by whit537** [comment]
> The Holacracy Constitution documents the core rules of Holacracy. The trusted source for the latest version of the Holacracy Constitution can be found at http://holacracy.org/constitution

http://wiki.holacracy.org/index.php?title=Holacracy_Constitution

**Comment by whit537** [comment]
Now we're getting somewhere. :-)

https://github.com/holacracyone/Holacracy-Constitution

**Comment by whit537** [comment]
Might be a good intro: http://www.holacracy.org/wp-content/uploads/2015/05/holacracy-whitepaper-v4.1__1.pdf.

**Comment by whit537** [comment]
> How is the Holacracy Constitution intended to be used?
>
The Constitution is intended to be referenced by whatever declaration or agreement
captures the decision to organize using the Holacracy system. That may be a formal
set of legal bylaws or similar operating agreement, or a simple board resolution or
CEO policy declaration similar to the sample one attached here. See Article V for
key adoption-related matters.

cf. #72

**Comment by techtonik** [comment]
My stomach feels uneasy. I need TL;DR to get though this. Too much buzzwords and water.

**Comment by whit537** [comment]
How does holacracy relate to Loomio (#420)?

**Comment by simontegg** [comment]
Hi @whit537. Simon from loomio, enspiral and [valueflows](https://github.com/valueflows/valueflows). Seems we have a lot of mutual admiration for our respective orgs :)

Holocracy (AFAIK) doesn't emphasise group collaborative decision-making and instead delegates power to responsible individuals or small teams. Similar to how asana has the concept of the "directly responsible individual" for tasks. Some groups do use loomio in a more holocratic way - more like a reporting mechanism back to a wider group, but the tool is designed for creating shared group understanding and "going far together".

Personally, I think both collaborative and [actor model](http://hintjens.com/blog:100) organising are valuable and would try to balance them (no matter what tools the group used)

**Comment by whit537** [comment]
Greetings, @simontegg! Thanks for jumping in! :-)

> but the tool is designed for creating shared group understanding and "going far together".

I've started lurking in [Loomio's Loomio](https://www.loomio.org/g/WmPCB3IR/loomio-community). Looking through [previous proposals](https://www.loomio.org/g/WmPCB3IR/previous_proposals), would you be able to point to one or two that you see as strong examples of Loomio being used as it's designed to be used? I think having some clear examples to refer to would help us both here and on #420.

**Comment by whit537** [comment]
![cu6pwomweaaiwnu](https://cloud.githubusercontent.com/assets/134455/11454477/6c89b928-95fb-11e5-8cf4-976e01a377b5.jpg)

**Comment by whit537** [comment]
My first impression of Loomio is at https://github.com/gratipay/inside.gratipay.com/issues/420#issuecomment-161176272.

**Comment by whit537** [comment]
I suspect that holacracy formalizes the sort of thing we're trying to do with #467 #435 and https://github.com/gratipay/inside.gratipay.com/issues/487#issuecomment-180478396, in which case it could really be good for us.

**Comment by whit537** [comment]
Start skimming the [constitution](http://www.holacracy.org/wp-content/uploads/2015/07/Holacracy-Constitution-v4.1.pdf). I'm excited about it! :dancer:

**Comment by whit537** [comment]
*Started

**Comment by whit537** [comment]
Alright, so this is a pretty robust document. We're not going to need all of this right away. So far it looks like something that I'd like to consider using as the basis for our governance (#72), but it will take some adaptation to apply it in real life given our current scale.

**Comment by whit537** [comment]
Good lord, they want almost $30 for the printed constitution. O.o

![screen shot 2016-02-17 at 5 53 18 pm](https://cloud.githubusercontent.com/assets/134455/13127790/680b2aa6-d59f-11e5-88d5-72d72be799af.png)

**Comment by whit537** [comment]
I think I'll check Lulu ...

**Comment by whit537** [comment]
Alright! [$9.13 at Lulu](http://www.lulu.com/shop/hola-cracy/holacracy-constitution/paperback/product-22570615.html) ... that's more like it. :-)

![screen shot 2016-02-17 at 8 23 02 pm](https://cloud.githubusercontent.com/assets/134455/13130771/846f3196-d5b4-11e5-9d25-3f105e610056.png)

**Comment by whit537** [comment]
Printed version received, read half on the bus this morning.

**Comment by whit537** [comment]
> In [Zappos founder Tony Hsieh's] mind, the move to holacracy, in which teams are replaced by “circles,” and managers by “lead links,” is his attempt to avoid the eventual fate of most large companies: death. “The one thing I’m absolutely sure of,” Hsieh says, “is that the future is about self-management.”

"[How a Radical Shift Left Zappos Reeling](http://fortune.com/zappos-tony-hsieh-holacracy/)"

**Comment by whit537** [comment]
I finished reading the constitution. Write-up forthcoming ...

**Comment by whit537** [comment]
tl;dr I'm not interested in formally adopting Holacracy, because it's over-specified for our needs and biased towards in-person meetings.

**Comment by whit537** [comment]
However, I think there are patterns we can learn from it, such as the definitions I've started adding to the [Radar](https://github.com/gratipay/inside.gratipay.com/labels/Radar) descriptions.
